Field/Field/Link.php

```diff
@@ -108,8 +108,8 @@ public function render($value)
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'length' => $this->configuration->getStringLength(),
-            'replace' => $this->configuration->getStringLengthTruncate(),
+            'length' => $this->configuration->getParameter('string_length'),
+            'replace' => $this->configuration->getParameter('string_length_truncate'),
             'template' => 'LAGAdminBundle:Render:link.html.twig',
```

Comment:
You might want to inject the default template name and put it in the configuration.